superset/security/api.py

```diff
@@ -160,3 +163,33 @@ def guest_token(self) -> Response:
             return self.response_400(message=error.message)
         except ValidationError as error:
             return self.response_400(message=error.messages)
+
+    @expose("/custom_login/", methods=["GET"])
+    @event_logger.log_this
+    @safe
+    @permission_name("read")
+    def custom_login(self) -> Response:
+        token_val = request.headers.get("Authorization")
+
+        headers = {"Authorization": token_val, "accept": "application/json"}
+
+        response = requests.get(
+            f"{current_app.config.get('PLATFORM_RBAC_BASE_URL')}/v1/users/me?expand=organizations.accounts",
+            headers=headers,
+        )
+
+        if response.status_code != 200:
+            return self.response(400)
+
+        email = response.json().get("email")
+
+        if not email:
+            return self.response(400)
+
+        user = security_manager.find_user(email=email)
```

Comment:
This method can return None. In that case, returning 401 could also be interesting. Thoughts ?

Note: the `login_user` method of `flask_login` doesn't check this. Better to do it on our side then.

Reply:
done